### ISSUE TITLE
We're Moving! - Docs to SiteNow

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#564]
- [x] Stop adding documentation to gh-pages. All new docs should be put on the new sitenow site.
- [x] Re-organize existing documentation.
- [x] Manually move docs to SiteNow site as pages.
  - [x] Create meaningful tags (e.g. WYSIWYG, Content Blocks, Site Configuration, Media) and tag pages.
  - [x] Minimize the use of images if possible.
  - [x] Minimize the use of accordions.
- [x] Update tours to point to new permalinks (https://sitenow.uiowa.edu/node/{nid})
- [x] Update default content to point to new documentation landing page /docs
- [ ] Query sites with existing default content and update links?
- [x] Update SiteNow Help menu link to point to new site.
- [x] Link to SiteNow 1 help and support articles.
- [x] Update gh-pages to announce the move
- [ ] Meta refresh redirects.
- [x] ???
- [ ] Profit!

**Comment by dnkemper** [comment]
Added all of the gh-pages docs to sitenow

**Comment by joewhitsitt** [comment]
Nice @dnkemper! Thanks!

**Comment by joewhitsitt** [comment]
@dnkemper and @ethanstjohn i'd be interested in hearing some feedback about the process at the next dev meeting. How did it go? Any noticeable issues/would be nice if type stuff.

**Comment by dnkemper** [comment]
## Does it matter that the user will see this on their screen when the *next* button is clicked during the tours? 
<img width="1042" alt="Page CT Section Settings (8:16)" src="https://user-images.githubusercontent.com/16872949/65692601-73f90200-e038-11e9-8f7b-9ebb629ef752.png">
Page CT Section Settings (8/16)- can't see tour box ^^

**Comment by dnkemper** [comment]
## Additional-Information(10of11)
<img width="400" alt="Additional-Information(10of11)" src="https://user-images.githubusercontent.com/16872949/65692603-73f90200-e038-11e9-96ae-619e43faa799.png">

**Comment by dnkemper** [comment]
## Footer-Links(9of11)
<img width="425" alt="Footer-Links(9of11)" src="https://user-images.githubusercontent.com/16872949/65692604-73f90200-e038-11e9-94ac-288762b1e839.png">

**Comment by dnkemper** [comment]
## Footer(5of11)

<img width="384" alt="Footer(5of11)" src="https://user-images.githubusercontent.com/16872949/65692605-73f90200-e038-11e9-8192-d316b8a2dd72.png">

**Comment by dnkemper** [comment]
## Additional Questions
@joewhitsitt @richardbporter @briand44 @GaryRidgway 
1. When adding an article, and adding a new author, you’re only allowed to enter a first and last name… should we set it up so we can add all of the other fields or leave it as is? 
2. Some pages are long with content. If user is looking for something specific ie Domain Changes and has to scroll throughout the page… is there a way we could have a skeleton outline of the page and the user can select what they want and it takes it to that section?
3. If you go to https://sitenow.prod.drupal.uiowa.edu/documentation/content-documentation/webforms there are multiple pages going in depth about the same topic. When you select a page, (for example, Advanced Webforms Config), the side menu disappears. Could we keep the sidebar menu and have the selected page highlighted? To show the other pages on the topic?  Or have previous/next buttons maybe? Would that make it more user friendly?
4. Should we have a 'Scroll to the top' option on pages? When I was looking at the webforms page on my phone, it was rough having to go back to the top of the page
5. Probably way too late now… but if we want users to pay for a customized website, should we maybe have our SiteNow site show what could be done if they decide to customize their site? Make it desirable = maybe more profit?

**Comment by joewhitsitt** [comment]
Thanks for this feedback @dnkemper. Looks like there is a z-index issue with our tours and the toolbar that I thought we had fixed a while back. This should be a bug issue.

I am aware that all of our ckeditor wysiwyg tour things are hit and miss based on how quickly ckeditor loads... If the target isn't there when the tour initiates, the tour doesn't have anything to attach to. This could be a bug worth filing, but a low-priority one at that. Documentation can make up for it.

Regarding your other comments:

1. Since the user is there to create an article, I think keeping the person fields to a minimum is a good idea. They can go fill in the user when they are done with the article.

2. We have an issue for a table of contents type paragraph #140, but I am not sure if we can easily accomplish it without technical work from the user (providing unique ids, titles, etc.).

3. Active trail menu is by design to keep menus from constantly being longer than the content. The breadcrumb navigation is the secondary aid to navigate back. Previous/Next buttons sounds like the book module which I think we want to avoid as I think it is being removed in D9/10.

4. I think this is a nice enhancement that we have used a lot of D7 sites. We could add an issue for this.

5. When we start working on custom sites, we can definitely highlight our work. Though HR is custom, it will be on the legacy theme once #371 is done and rolled out.

Thanks again for this user testing.
